libunwind/src/DwarfInstructions.hpp

```diff
@@ -54,6 +55,16 @@ class DwarfInstructions {
   typedef typename CFI_Parser<A>::FDE_Info          FDE_Info;
   typedef typename CFI_Parser<A>::CIE_Info          CIE_Info;
 
+#if defined(__CHERI_PURE_CAPABILITY__) && defined(_LIBUNWIND_SANDBOX_OTYPES)
+  static size_t restoreCalleeSavedRegisters(pint_t csp, A &addressSpace,
+                                            R &newRegisters,
+                                            CompartmentInfo &CI, pint_t sealer);
+  static pint_t restoreRegistersFromSandbox(pint_t csp, A &addressSpace,
+                                            R &newRegisters,
+                                            CompartmentInfo &CI, pint_t sealer);
+  static bool isTrampoline(pint_t ecsp, A &addressSpace, CompartmentInfo &CI,
```

Comment:
There are other types of trampolines in the world. Name should be specific in it being a compartment transition trampoline.